CHANGELOG.md

```diff
@@ -73,6 +73,11 @@
 - Added support for `DataFrame.corr`.
 - Added support for `DataFrame.equals` and `Series.equals`.
 - Added support for `DataFrame.reindex` and `Series.reindex`.
+- Added support for `Index.T`.
+- Added support for `Index.astype`.
+- Added support for `Index.unique` and `Index.nunique`.
+- Added support for `Index.value_counts`.
```

Comment:
new release have been made, so please add the new change into the new merges

Reply:
I only moved the value_counts log, I left the others in the 1.20 release since they were merged before the release.